### PR TITLE
Enable video transmitter tab

### LINKED ISSUE #3504: VTX configuration is not possible
### Describe the bug

`Video Transmitter` tab disappears and VTX config is not possible when `VTX (MSP+Displayport)` configuration is set on ports tab.

### To Reproduce

Enable `VTX (MSP+Displayport)` config.

### Expected behavior

Video Transmitter tab is shown and VTX configuration is possible when VTX MSP is used. 

### Configurator version

10.10.0-debug-9b9e5bf

### Flight controller configuration

```txt
# feature
feature TELEMETRY
feature OSD

# serial
serial 0 64 115200 57600 0 115200
serial 6 131073 115200 57600 0 115200

# master
set acc_calibration = -4,-40,26,1
set osd_warnings_pos = 14740
set osd_canvas_width = 30
set osd_canvas_height = 13
```

### Add any other context about the problem that you think might be relevant here

`Video Transmitter` tab is behaving correctly if `TBS Smartaudio` or `Tramp` proto is selected. 
Newest configuration hides the VTX tab even for working betaflight-4.4 versions (which are working correctly with Configurator 10.9.0).

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes #3504

- Fixes backwards compatibility
- Enables virtual vtx feature if vtx build option is present independent of port settings.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by cruwaller** [comment]
Just tested and still no `Video Transmitter` tab. 
There is a warning on `Ports` tab: 

> The VTX table has not been set up correctly and without it VTX control will not be possible. Please set up the VTX table in Video Transmitter tab.

quite funny since there are no such a tab :D 

Anyway, VTX table is set and works correctly :) 

```
# vtxtable
vtxtable bands 5
vtxtable channels 8
vtxtable band 1 BOSCAM_A A FACTORY    0    0    0    0    0    0    0    0
vtxtable band 2 BOSCAM_B B FACTORY    0    0    0    0    0    0    0    0
vtxtable band 3 BOSCAM_E E FACTORY    0    0    0    0    0    0    0    0
vtxtable band 4 FATSHARK F FACTORY    0 5760    0 5800    0    0    0    0
vtxtable band 5 RACEBAND R FACTORY 5658 5695 5732 5769 5806 5843 5880 5917
vtxtable powerlevels 3
vtxtable powervalues 14 23 0
vtxtable powerlabels 25 200 0
```

**Comment by haslinghuis** [comment]
Did you add VTX in Other Options while flashing?

**Comment by cruwaller** [comment]
Seriously? As mentioned earlier... Quad works and flies, even the VTX control works as expected :) 
I test both 4.4 and latest 4.5 Betaflight versions (two different quads)...

`Video Transmitter` tab is shown and works correctly with 10.9.0 Configurator version... even the VTX settings can be adjusted correctly with BF4.5.

**Comment by haslinghuis** [comment]
Then why does it work here:

image

Going to have to remove the hiding feature

**Comment by haslinghuis** [comment]
Added new version

**Comment by cruwaller** [comment]
Just tested the latest version and VTX tab is missing also for SA and Tramp + still no tab for MSP.

2023-07-08_11-01_no_video_transmitter_tab

**Comment by haslinghuis** [comment]
Need to fix backwards compatibility as the virtual vtx feature only works with 4.5.0

**Comment by cruwaller** [comment]
Now the VTX tab is shown for both 4.5 and 4.4. Thanks a lot!

**Comment by haslinghuis** [comment]
Thanks for testing :+1:

**Review by cruwaller — APPROVED** [review]

**Comment by haslinghuis** [comment]
Rebased

**Review by blckmn — APPROVED** [review]

**Review by freasy — APPROVED** [review]
LGTM

**Comment by hydra** [comment]
This is still broken for me.

image

image

My use case is an ELRS CRSF RX with built-in SPI-connected VTX.

VTX control seems to work as expected after using the cli to configure vtx tables, but the configurator does not show the `Video Transmitter` tab.

**Comment by cruwaller** [comment]
I faced this same issue again with the latest nightly build yesterday (on linux). Unfortunately that VTX tab handling seems to be still a bit unstable... `USE_VTX` is still enabled and cli works.
I will retest with #3512 branch.

**Comment by haslinghuis** [comment]
@cruwaller removed the logic as a whole as more code would be required to support, rather just show the tab.

**Comment by haslinghuis** [comment]
@cruwaller new version in # 3512 should work now regardless

**Pull request merged**
